packages/strapi-plugin-settings-manager/services/SettingsManager.js

```diff
@@ -450,6 +450,34 @@ module.exports = {
             target: 'server.cron.enabled',
             type: 'boolean',
             value: _.get(strapi.config, `environments.${env}.server.cron.enabled`, null)
+          },
+          {
```

Comment:
Can you create a new section for Proxy settings please.
Like [here](https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-settings-manager/services/SettingsManager.js#L219)